site/docs/relations/logical_relations.md

```diff
@@ -199,7 +199,8 @@ The cross product operation will combine two separate inputs into a single outpu
 | Inputs               | 2                                                            |
 | Outputs              | 1                                                            |
 | Property Maintenance | Distribution is maintained. Orderedness is empty post operation. |
-| Direct Output Order  | The emit order of the left input followed by the emit order of the right input. |
+| Input Order          | The input order is the left input followed by the emit order of the right input. All field references of [Cross Product Properties](#cross-product-properties) are over this order. |
```

Comment:
```suggestion
| Input Order          | The input order is the left input followed by the the right input. All field references of [Cross Product Properties](#cross-product-properties) are over this order. |
```

Nit: minor inconsistency between left & right

Let's avoid `emit order` when defining `Input Order`.  Or we could say `emit order of the left input followed by emit order of the right input` but I think "emit has been applied" is implicit at this point so we can just leave it out.